src/components/AircrackNG/AircrackNG.tsx

```diff
@@ -237,29 +247,56 @@ const AircrackNG = () => {
+                        onChange={(e) => setSelectedModeOption(e.target.value)}
+                        data={types}
+                        required
+                        placeholder={"Pick a attack mode"}
+                        label={"WEP or WPA-PSK"}
+                    />
                     <Switch
                         size="md"
                         label="Advanced Mode"
                         checked={advanceMode}
                         onChange={(e) => setAdvanceMode(e.currentTarget.checked)}
                     />
-                    <Switch
-                        size="md"
-                        label="Custom Configuration"
-                        checked={customMode}
-                        onChange={(e) => setCustomMode(e.currentTarget.checked)}
+                    <TextInput
+                        label={"Path to wordlist(s) filename(s)."}
+                        required
+                        {...form.getInputProps("wordList")}
+                        {...form.getInputProps("filePath")}
+                    />
+                    {isWEP && (
+                        <TextInput
+                            label={"Set AP MAC address"}
+                            placeholder={"eg: xx:xx:xx:xx:xx:xx"}
+                            required
+                            {...form.getInputProps("BSSID")}
+                        />
+                    )}
+                    {isWPA && <TextInput label={"Set AP identifier"} required {...form.getInputProps("ESSID")} />}
+                    {isWPA && (
+                        <TextInput
+                            label={"Use Pcap File (Please Supply FilePath and filename )"}
+                            placeholder={"eg: x/x/*.cap"}
+                            required
+                            {...form.getInputProps("filePath")}
+                        />
+                    )}
+                    <TextInput
+                        label={"Save as key to output file (Please Supply FilePath)"}
+                        required
+                        {...form.getInputProps("filePath")}
                     />
-                    <TextInput label={"Name of your fake host"} required {...form.getInputProps("fakeHost")} />
-                    <TextInput label={"Channel of choice"} required {...form.getInputProps("channel")} />
-                    <TextInput label={"Your WLAN interface"} required {...form.getInputProps("replayInterface")} />
-                    {advanceMode && (
-                        <>
-                            <TextInput label={"Set AP MAC address"} {...form.getInputProps("MACAddress")} />
-                            <TextInput
-                                label={"Save as Pcap File (Please Supply FilePath)"}
-                                {...form.getInputProps("filePath")}
-                            />
-                        </>
+                    {advanceMode && isWEP && (
```

Comment:
Nothing appears to happen when I switch on advanced mode after selecting WPA. Is this intended? 

If this is intended, I wonder if it better to hide the advanced mode switch until somebody selects WEP. 

![image](https://github.com/user-attachments/assets/4ff20d49-9b7e-4bc5-91f1-fa0f846f7b8d)